cmd/podmanV2/pods/kill.go

```diff
@@ -0,0 +1,68 @@
+package pods
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/containers/libpod/cmd/podmanV2/parse"
+	"github.com/containers/libpod/cmd/podmanV2/registry"
+	"github.com/containers/libpod/cmd/podmanV2/utils"
+	"github.com/containers/libpod/pkg/domain/entities"
+	"github.com/spf13/cobra"
+)
+
+var (
+	podKillDescription = `Signals are sent to the main process of each container inside the specified pod.
+
+  The default signal is SIGKILL, or any signal specified with option --signal.`
+	killCommand = &cobra.Command{
+		Use:   "kill [flags] POD [POD...]",
+		Short: "Send the specified signal or SIGKILL to containers in pod",
+		Long:  podKillDescription,
+		RunE:  kill,
```

Comment:
Don't we need the persistentPreRunE?

Reply:
@mheon The parent pod command will do that.

Reply:
nope, inherited from the pods subcommand

Reply:
Is this documented anywhere? Because this seems somewhat unintuitive.